lib/dynflow/execution_plan/steps/abstract.rb

```diff
@@ -97,7 +100,10 @@ def to_hash
                           real_time:           real_time,
                           progress_done:       progress_done,
                           progress_weight:     progress_weight,
-                          queue:               queue
+                          queue:               queue,
+                          wakeup_info:         (wakeup_info && wakeup_info.collect do |info|
+                            { time: info.time, event_cls: info.event.name }
```

Comment:
What is event supposed to be here? No matter what I try to pass I'm getting

```
2019-08-21T16:50:08.044Z 1 TID-gs512uf31 WARN: NoMethodError: undefined method `name' for nil:NilClass
2019-08-21T16:50:08.044Z 1 TID-gs512uf31 WARN: /data/lib/dynflow/execution_plan/steps/abstract.rb:106:in `block in to_hash'
/data/lib/dynflow/execution_plan/steps/abstract.rb:105:in `collect'
/data/lib/dynflow/execution_plan/steps/abstract.rb:105:in `to_hash'
/data/lib/dynflow/director.rb:70:in `to_hash'
/data/lib/dynflow/serializer.rb:38:in `generate_other'
/usr/share/gems/gems/algebrick-0.7.5/lib/algebrick/serializer.rb:41:in `dump'
/data/lib/dynflow/executors/sidekiq/serialization.rb:8:in `serialize'
/data/lib/dynflow/executors/sidekiq/serialization.rb:20:in `block in client_push'
/data/lib/dynflow/executors/sidekiq/serialization.rb:20:in `map'
/data/lib/dynflow/executors/sidekiq/serialization.rb:20:in `client_push'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/worker.rb:93:in `perform_async'
/data/lib/dynflow/executors/sidekiq/worker_jobs.rb:17:in `ensure in perform'
/data/lib/dynflow/executors/sidekiq/worker_jobs.rb:17:in `perform'
/data/lib/dynflow/executors/sidekiq/serialization.rb:27:in `perform'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:192:in `execute_job'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:165:in `block (2 levels) in process'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/middleware/chain.rb:128:in `block in invoke'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/middleware/chain.rb:133:in `invoke'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:164:in `block in process'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:137:in `block (6 levels) in dispatch'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/job_retry.rb:109:in `local'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:136:in `block (5 levels) in dispatch'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq.rb:37:in `block in <module:Sidekiq>'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:132:in `block (4 levels) in dispatch'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:250:in `stats'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:127:in `block (3 levels) in dispatch'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/job_logger.rb:8:in `call'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:126:in `block (2 levels) in dispatch'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/job_retry.rb:74:in `global'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:125:in `block in dispatch'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/logging.rb:48:in `with_context'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/logging.rb:42:in `with_job_hash_context'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:124:in `dispatch'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:163:in `process'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:83:in `process_one'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/processor.rb:71:in `run'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/util.rb:16:in `watchdog'
/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/util.rb:25:in `block in safe_thread'
```

where receiver of `name` is the event I tried to send through the clock

Reply:
it's passed from clock's `with_what` and should be event class, but probably make sense as nil as well.